amethyst_core/src/transform/components/transform.rs

```diff
@@ -673,6 +696,10 @@ impl<'de> Deserialize<'de> for Transform {
                     .next_element()?
                     .ok_or_else(|| de::Error::invalid_length(2, &self))?;
 
+                let parent_relation: ParentTransformRelation = seq
+                    .next_element()?
+                    .map_or_else(|| ParentTransformRelation::Relative, |v| v);
```

Comment:
does `|v| v` do anything? (would `unwrap_or_else` be more apt?)
also, there's a `.copied()` in case the mapping copies the value

Reply:
Done and updated! Dunno why I did that, I must be tired.